### PR TITLE
[13.0][MIG] attachment_unindex_content: Migration to 13.0

### PR DESCRIPTION
**Pull request by randall-vx**

### CODE REVIEW

**Comment by randall-vx** [comment]
@moylop260 @luisg123v ready for review.

regards,

File: excel_import_export/models/xlsx_template.py [hunk 1]
```diff
@@ -208,10 +208,9 @@ def _update_result_field_common_wizard(self):
             _field.ensure_one()
             _field.write({"relation": self.result_model_id.model})
         _field.compute = """
-self['%s'] = self.env['%s'].search(self.safe_domain(self.domain))
-        """ % (
-            self.result_field,
-            self.result_model_id.model,
+self['{}'] = self.env['{}'].search(self.safe_domain(self.domain))
+        """.format(
```
Comment (luisg123v) [inline excel_import_export/models/xlsx_template.py:212]: This is an unrelated module.

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: excel_import_export/models/xlsx_template.py (hunk 1 above)
Comment (randall-vx) [reply excel_import_export/models/xlsx_template.py:212]: @luisg123v  this change was made by the pre-commit stage. Without this change the pre-commit does not pass to build.
